app/styles/app.css

```diff
@@ -1178,6 +1178,8 @@ input#search:-ms-input-placeholder {
 body * { visibility: hidden; }
 #printcontent * { visibility: visible; }
 #printcontent { position: absolute; top: 100px; left: 100px; width: 160%;height: 100%;}
+    #printreceiptcontent * { visibility: visible; }
+    #printreceiptcontent { position: absolute; top: 100px; left: 100px; width: 160%;height: 100%;}
```

Comment:
I tried using the existing one but it was giving duplicate id error so created similar one and used it .